src/Bigtable/Connection/TableConnectionInterface.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\ConnectionInterface;
+
+/**
+ * Connection to Bigtable Table API
+ */
+interface TableConnectionInterface extends ConnectionInterface
+{
+
+    /**
+     * @param string $tableId
+     *
+     * @param Entry[] $entries      The row keys and corresponding mutations to be applied in bulk.
+     *                              Each entry is applied as an atomic mutation, but the entries may be
+     *                              applied in arbitrary order (even between entries for the same row).
+     *                              At least one entry must be specified, and in total the entries can
+     *                              contain at most 100000 mutations.
+     * @param array   $optionalArgs {
+     *                              Optional.
+     *
+     *     @type int $timeoutMillis
+     *          Timeout to use for this call.
+     * }
+     */
+    public function mutateRows($tableId, array $entries, array $optionalArgs = []);
+
+    /**
+     * @param string $tableId
+     *
+     * @param string     $rowKey       The key of the row to which the mutation should be applied.
+     *
+     * @param Mutation[] $mutations    Changes to be atomically applied to the specified row. Entries are applied
```

Comment:
What is a `Mutation` object?  Are we going to provide a wrapper around the protobuf, or is this the protobuf itself?

Reply:
As brought up [in the past review](https://github.com/GoogleCloudPlatform/google-cloud-php/pull/930#pullrequestreview-105471541) this should be a wrapper around protobuf. It would be great if we could stick to the standards already set in the library so as to not create confusion for our users.

Reply:
+1.  This object should be a RowMutation.  [Here's the java version](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java).